spec/components/psul_constraint_layout_component_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PsulConstraintLayoutComponent, type: :component do
+  subject(:rendered) do
+    render_inline(described_class.new(**params))
+  end
+
+  describe 'for simple display' do
+    let(:params) do
+      { label: 'my label', value: 'my value' }
+    end
+
+    it 'renders label and value' do
+      expect(rendered).to have_selector('span.applied-filter.constraint') do |s|
+        expect(s).to have_css('span.constraint-value')
```

Comment:
I don't think this block is doing what you think it's doing. I believe what is happening is that after line 16, the expect statement evaluates and the block is ignored. If you put in `expect(s).to have_css('span.garbage-value')` it will still pass.

The easiest thing to do would just be to put them all inline:

```ruby
      expect(rendered).to have_selector('span.applied-filter.constraint')                                                  
      expect(rendered).to have_css('span.constraint-value')                                                                
      expect(rendered).not_to have_css('a.constraint-value')                                                               
      expect(rendered).not_to have_css('span.btn.btn-outline-secondary.constraint-value')                                  
      expect(rendered).to have_selector 'span.filter-name', text: 'my label'                                               
      expect(rendered).to have_selector 'span.filter-value', text: 'my value'
```

I was looking around for something like `within('span.applied-filter.constraint') do` which is what we do in feature tests, but rendered would need to be a `Capybara::Node::Simple` or something. Probably not worth getting into now, but something we could refactor later.

Reply:
ah yeah thanks for catching this, I had it as Capybara::Node::Simple then removed it, I will put that back and check

Reply:
hah I thought 
```
subject(:rendered) do
    Capybara::Node::Simple.new(render_inline(described_class.new(**params)))
end
```
would work but same behavior 🤔  I mean it passes for any value.

guess I will just put them inline as you suggested, unless you have suggestion for how to get `Capybara::Node::Simple` to work.

Reply:
I'd say leave it for now and we can revisit later.

Reply:
yup, just converted them to inline 👍